packages/databricks-vscode/src/configuration/AzureCliCheck.ts

```diff
@@ -12,6 +12,12 @@ import {AzureCliAuthProvider} from "./auth/AuthProvider";
 const extensionVersion = require("../../package.json")
     .version as ProductVersion;
 
+type AzureCloud =
+    | "AzureCloud"
+    | "AzureChinaCloud"
+    | "AzureUSGovernment"
+    | "AzureGermanCloud";
```

Comment:
Last enum looks unused.

Reply:
Actually let me check what the German URLs are before merging

Reply:
We are not in Azure Germany. Took out this enum value.